src/engraving/libmscore/staff.cpp

```diff
@@ -816,6 +816,9 @@ void Staff::write(XmlWriter& xml) const
     if (_mergeMatchingRests) {
         xml.tag("mergeMatchingRests", _mergeMatchingRests);
     }
+    if (!visible()) {
+        xml.tag("isStaffVisible", visible());
```

Comment:
Can we use this tag instead?
xml.tag("invisible", isLinesInvisible(Fraction(0, 1)));